### ISSUE TITLE
No license file present so compliance to indicated 3-clause BSD isn't possible

### ISSUE THREAD

**Issue by rowanthorpe** [norswap/rocketcrash#1]
Your README indicates that rocketcrash is licensed under the [BSD 3-clause license](https://opensource.org/licenses/BSD-3-Clause), and the template linked to there shows that one of the clauses requires copying your copyright notice, but you don't include the actual copyright/license info (in a separate file or as headers in any of the existing files), so it is not presently possible to comply with the indicated (but absent) license. I am considering using rocketcrash in a project (thanks for creating it!), but this issue is a blocker for doing so.

**Comment by norswap** [comment]
Fixed!

I'd have taken care of this earlier, but apparently Github's not sending me mail notifications for issues on my repos.

Let me know if you encounter any issues, as this as not been extensively used, as far as I know.

**Comment by rowanthorpe** [comment]
Thanks! I will let you know as/when I find any issues. However, in the interests of prudence I am deliberately using it minimally (just for jumping to exit/cleanup, to make my code less verbose). The same minimalism is the reason I like your lib in the first place - I was able to grok it in one reading, and can reason about what is really going on (as opposed to most "big" exception libraries, which become too much of an exercise in trust for my liking).

So far (after I made two tiny tweaks which I will PR to you soon) the lib has worked fine, even under extreme compiler-warning flags, and rigorous valgrind testing. I also made a few syntactic tweaks for it to obey the really pedantic GCC compiler warnings - if you want me to also PR those I can (let me know), but I know some maintainers find that more annoying than helpful...
